include/lsst/jointcal/MeasuredStar.h

```diff
@@ -41,8 +41,13 @@ class MeasuredStar : public BaseStar {
               _instMag(0.),
               _instMagErr(0.) {}
 
-    /// No move, allow copy constructor: we may copy the fitted StarLists when associating and matching
-    /// catalogs, otherwise Stars should be managed by shared_ptr only.
+    MeasuredStar(double xx, double yy, double flux, double fluxErr, double instFlux, double instFluxErr)
+            : MeasuredStar(BaseStar(xx, yy, flux, fluxErr)) {
+        setInstFluxAndErr(instFlux, instFluxErr);
+    }
```

Comment:
I've deleted this constructor: turns out I don't really need it.

BaseStar is any kind of star (Measured, Fitted, Reference), and only the former has `instFlux` (since it's from a SourceCatalog measurement).